v2/internal/project/project.go

```diff
@@ -85,6 +86,16 @@ type Project struct {
 	NSISType string `json:"nsisType"`
 }
 
+func (p *Project) GetDevBuildCommand() string {
+	if p.DevBuildCommand != "" {
+		return p.DevBuildCommand
+	}
+	if p.DevCommand != "" {
+		return p.DevCommand
```

Comment:
Do you think it might be worth doing a `println("Warning: frontend:dev has been deprecated, please use frontend:dev:build")`? I think the templates will need updating too.

Reply:
We can do that at a later date 👍